Yoast/javascript
# Add Czech researcher to getResearcher helper

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds Czech Researcher to `getResearcher` helper so that we can load Czech to the content-analysis app. Also deletes flesch reading ease research from the Researcher as it's not supported in Czech.

## Relevant technical choices:

* The premium-configuration branch with the same name was used to add a missing change to the Czech morphologyData file and should be CRed and merged as well.

## Test instructions

This PR can be tested by following these steps:

In content-analysis app:
* Add text minimum 300 words.
* Set the language to `cz`
* Confirm that the Czech SEO and readability analysis load
* Set a keyphrase, add it to the text and see that it's recognized (test also with different forms of a word)
* Change some readability metrics (e.g., add transition words) and confirm that the analysis updates accordingly.
* Confirm that there is no error on the console.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-695
